### PR TITLE
Issue #3209: Adding color schemes to Basis and adding a few new color…

### PR DESCRIPTION
**Pull request by hosef**
… options to handle some cases where colors were too close together.

### CODE REVIEW

File: core/themes/basis/color/color.inc [hunk 1]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'borders' => '#24272c',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#888889',
+        'primarytabstext' => '#d0d0d0',
+        'buttons' => '#014958',
+
+        'footer' => '#d0d0d2',
+        'footertext' => '#000100',
+        'footerborder' => '#24272c',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: With the footer background white, we can make the top-border a much softer gray, maybe `#C5C5C5` (same as header background).

Maybe that would stick with the subtle feel of the ice theme :)

File: core/themes/basis/color/color.inc [hunk 2]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'text' => '#2a2a2a',
+        'link' => '#00809c',
+        'borders' => '#24272c',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#888889',
+        'primarytabstext' => '#d0d0d0',
+        'buttons' => '#014958',
+
+        'footer' => '#d0d0d2',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: I'm worried the dark blue text won't be readable on the dark gray background. Can we leave the footer background white? `#ffffff`

File: core/themes/basis/color/color.inc [hunk 3]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'borders' => '#2c2c28',
+        'formfocusborder' => '#edede7',
+
+        'primarytabs' => '#2b2b2b',
+        'primarytabstext' => '#301313',
+        'buttons' => '#c9d9e3',
+
+        'footer' => '#e7e7e7',
+        'footertext' => '#000100',
+        'footerborder' => '#2c2c28',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: Can we make the footer border purple? How about `#89297A`?

File: core/themes/basis/color/color.inc [hunk 4]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'link' => '#c63d0f',
+        'borders' => '#dec0b4',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#2b2b2b',
+        'primarytabstext' => '#3b3738',
+        'buttons' => '#dec0b4',
+
+        'footer' => '#7e8f7c',
+        'footertext' => '#000100',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: I don't think the dark text is readable on the dark background. Can we change footer text color to `#CFD7B2`?

File: core/themes/basis/color/color.inc [hunk 5]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'link' => '#fa6900',
+        'borders' => '#4c5036',
+        'formfocusborder' => '#f3f4e9',
+
+        'primarytabs' => '#eda766',
+        'primarytabstext' => '#e0e4cc',
+        'buttons' => '#4c5038',
+
+        'footer' => '#798054',
+        'footertext' => '#000100',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc:124]: I don't think the dark text is readable on the dark background. Can we change footer text color to `#CFD7B2`?

File: core/themes/basis/color/color.inc [hunk 6]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'borders' => '#dec0b4',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#2b2b2b',
+        'primarytabstext' => '#3b3738',
+        'buttons' => '#dec0b4',
+
+        'footer' => '#7e8f7c',
+        'footertext' => '#000100',
+        'footerborder' => '#dec0b4',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: Pink on green, euh. Can we make the border color the same as the footer background? `#6B7D68`

File: core/themes/basis/color/color.inc (hunk 1 above)
Comment (jenlampton) [reply core/themes/basis/color/color.inc]: looks great!

File: core/themes/basis/color/color.inc (hunk 3 above)
Comment (jenlampton) [reply core/themes/basis/color/color.inc]: looks great now!

File: core/themes/basis/color/color.inc (hunk 2 above)
Comment (jenlampton) [reply core/themes/basis/color/color.inc]: looks great now!

File: core/themes/basis/color/color.inc (hunk 5 above)
Comment (jenlampton) [reply core/themes/basis/color/color.inc:124]: looks great now!

File: core/themes/basis/color/color.inc (hunk 6 above)
Comment (jenlampton) [reply core/themes/basis/color/color.inc]: looks great now!

Comment (jenlampton) [reply core/themes/basis/color/color.inc]: This border looks better. I'm still not sure people will be able to read either text or the orange link color in the footer for this color pallette, though.

**Comment by jenlampton** [comment]
I had a thought about the goldfish footer!  Going to test it in the sandbox...
edit: what about this: http://2613.backdrop.backdrop.qa.backdropcms.org

I also noticed the hover color on menu links makes them low contrast... fiddling...

File: core/themes/basis/color/color.inc [hunk 7]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'text' => '#3b3b3b',
+        'link' => '#fa6900',
+        'borders' => '#4c5036',
+        'formfocusborder' => '#f3f4e9',
+
+        'primarytabs' => '#eda766',
+        'primarytabstext' => '#e0e4cc',
+        'buttons' => '#4c5038',
+
+        'footer' => '#cfd7b2',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: how about `#F0F1E3` for footer background?

File: core/themes/basis/color/color.inc [hunk 8]
```diff
@@ -26,15 +32,122 @@ $info = array(
...
+        'footertext' => '#000100',
+        'footerborder' => '#4c5036',
```
Comment (jenlampton) [inline core/themes/basis/color/color.inc]: How about `#CFD7B2` for footer border?

**Comment by hosef** [comment]
@jenlampton I updated the PR with the color changes again.

**Comment by quicksketch** [comment]
Thanks! Glad to see this finally knocked out. I've only merged into 1.x for 1.13.0.

**Pull request merged**